mk/package-and-sign.sh

```diff
@@ -64,13 +64,8 @@ echo "INFO: Build and sign the installers..."
 . ${REPO}/mk/build-installers.sh
 
 #collect output and extra files to the OUTPUT_DIR
-EN_CD_DIR=${OUTPUT_DIR}/installer
-mkdir_clean ${EN_CD_DIR}
-cp ${DOTNETINST}/${BRANDING_BRAND_CONSOLE}Setup.exe ${EN_CD_DIR}
-cp ${REPO}/Branding/Images/AppIcon.ico ${EN_CD_DIR}/${BRANDING_BRAND_CONSOLE}.ico
-L10N_CD_DIR=${OUTPUT_DIR}/installer.l10n
-mkdir_clean ${L10N_CD_DIR}
-cp ${DOTNETINST}/${BRANDING_BRAND_CONSOLE}Setup.l10n.exe ${L10N_CD_DIR}
+cp ${DOTNETINST}/${BRANDING_BRAND_CONSOLE}Setup.exe ${OUTPUT_DIR}
```

Comment:
Yes; it can be easily done when we next bump the build number; it's not relevant before this time anyway.